### PR TITLE
docs: improve hyperlinks in magma docs

### PR DESCRIPTION
**Pull request by mpfirrmann**
## Summary

Many links in the docs were out of date, either showing a 404 error or being redirected to github.com / magma.github.to. Here, we improve these redirects.

## Test Plan

- Check the links.

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: docs/readmes/feg/debug_cli.md [hunk 1]
```diff
@@ -7,8 +7,7 @@ hide_title: true
 # FeG CLI
 
 Some Go services on FeG include a cli to run commands against the
-3GPP core. The different clients source code can be seen at
-[here](https://github.com/magma/magma/tree/master/feg/gateway/tools)
+3GPP core. The different clients source code can be seen [here](https://github.com/magma/magma/tree/master/feg/gateway/tools).
```
Comment (LKreutzer) [inline docs/readmes/feg/debug_cli.md]: Could we change this to "can be seen in the [`feg/gateway/tools`](...) folder"?
Current rule is that we do not use `here` for links - see [`docs`](https://github.com/magma/magma/wiki/Contributing-Documentation#styling-and-writing-conventions).

File: docs/readmes/howtos/inbound_roaming.md [hunk 2]
```diff
@@ -96,9 +96,9 @@ Before starting to configure roaming setup, first you need to bring up a
 setup to handle your own/local subscribers. So before configuring Inbound
 Roaming you need:
 
-- Install [Or8cr](https://docs.magmacore.org/docs/orc8r/architecture_overview),
-- Install [Federatetion Gateway](https://docs.magmacore.org/docs/feg/deploy_intro) and,
-- Install [Access Gateway](https://docs.magmacore.org/docs/lte/setup_deb).
+- Install [Or8cr](https://magma.github.io/magma/docs/orc8r/architecture_overview),
+- Install [Federatetion Gateway](https://magma.github.io/magma/docs/feg/deploy_intro) and,
```
Comment (LKreutzer) [inline docs/readmes/howtos/inbound_roaming.md]: Could we fix the typo please `Federation`?

File: docs/readmes/howtos/troubleshooting/user_unable_to_attach.md [hunk 3]
```diff
@@ -50,7 +50,7 @@ PROTOCOL_ERROR 111
 
 Suggested Solution:
 
-1. Identify which parameter the UE is sending in the Attach Request and compare with the ones Magma support. You can find which parameters magma is currently supported from the code. <https://github.com/magma/magma/blob/master/lte/gateway/c/oai/tasks/nas/emm/msg/AttachRequest.c>
+1. Identify which parameter the UE is sending in the Attach Request and compare with the ones Magma support. You can find which parameters magma is currently supported from the code. <https://github.com/magma/magma/blob/master/lte/gateway/c/oai/tasks/nas/emm/msg/AttachRequest.cpp>
```
Comment (LKreutzer) [inline docs/readmes/howtos/troubleshooting/user_unable_to_attach.md]: Hm, I don't know why but I cannot open this link?
Screenshot from 2023-01-25 16-33-59

File: docs/readmes/lte/integrated_5G_sa.md [hunk 4]
```diff
@@ -26,9 +26,9 @@ Following diagram gives an overview of the 5G SA components.
 
 Before starting to configure 5G SA setup, first you need to bring up a setup to handle your own/local subscribers. So before configuring Inbound Roaming you need:
 
-- Install [Or8cr](https://docs.magmacore.org/docs/orc8r/architecture_overview),
-- Install [Federatetion Gateway](https://docs.magmacore.org/docs/feg/deploy_intro) and,
-- Install [Access Gateway](https://docs.magmacore.org/docs/lte/setup_deb).
+- Install [Or8cr](https:///magma.github.io/magma/docs/orc8r/architecture_overview),
+- Install [Federatetion Gateway](https:///magma.github.io/magma/docs/feg/deploy_intro) and,
```
Comment (LKreutzer) [inline docs/readmes/lte/integrated_5G_sa.md]: Could we fix the typo please? `Federation`

File: docs/readmes/nms/alerts_troubleshooting.md [hunk 5]
```diff
@@ -28,12 +28,12 @@ It is one of the key KPI and may impact network deployment/expansion/operations
...
 
 1. If any Orchestrator pods has some issue, please try to resolve that first.
-2. Make sure that Gateways are successfully checking in. If they are not checked-in, means that their metrics won’t be reported. Please follow [these steps](https://docs.magmacore.org/docs/howtos/troubleshooting/agw_unable_to_checkin) to troubleshoot.
+2. Make sure that Gateways are successfully checking in. If they are not checked-in, means that their metrics won’t be reported. Please follow [these steps](https://magma.github.io/magma/docs/howtos/troubleshooting/agw_unable_to_checkin) to troubleshoot.
 3. Perform relevant sync/changes if any update has been performed on NMS -> Network.
-4. Make sure that configuration for [Gateway](https://docs.magmacore.org/docs/lte/deploy_config_agw), [eNodeB](https://docs.magmacore.org/docs/lte/deploy_config_enodebd) and [APN](https://docs.magmacore.org/docs/lte/deploy_config_apn)has been followed.
+4. Make sure that configuration for [Gateway](https://magma.github.io/magma/docs/lte/deploy_config_agw), [eNodeB](https://magma.github.io/magma/docs/lte/deploy_config_enodebd) and [APN](https://magma.github.io/magma/docs/lte/deploy_config_apn)has been followed.
```
Comment (LKreutzer) [inline docs/readmes/nms/alerts_troubleshooting.md]: Could we add a space before the `has been followed.`?

File: docs/readmes/nms/alerts_troubleshooting.md [hunk 6]
```diff
@@ -106,7 +106,7 @@ Network, eNodeB, Gateway, Subscribers.
 
 ### Description
 
-It was observed that AGW is not able to check-in to Orchestrator as described [here](https://docs.magmacore.org/docs/lte/deploy_config_agw).
+It was observed that AGW is not able to check-in to Orchestrator as described [here](https://magma.github.io/magma/docs/lte/deploy_config_agw).
```
Comment (LKreutzer) [inline docs/readmes/nms/alerts_troubleshooting.md]: Could we not use `here` here?

**Review by LKreutzer — CHANGES_REQUESTED** [review]

File: docs/readmes/howtos/troubleshooting/user_unable_to_attach.md (hunk 3 above)
Comment (voisey) [reply docs/readmes/howtos/troubleshooting/user_unable_to_attach.md]: Same

Comment (mpfirrmann) [reply docs/readmes/howtos/troubleshooting/user_unable_to_attach.md]: Should be fixed, thx.

File: docs/readmes/nms/alerts_troubleshooting.md (hunk 5 above)
Comment (LKreutzer) [reply docs/readmes/nms/alerts_troubleshooting.md]: I think the space is still missing between `/deploy_config_apn)has`?

Comment (mpfirrmann) [reply docs/readmes/nms/alerts_troubleshooting.md]: There were two and I fixed the one you didn't complain about ;) Both are fixed now.

**Review by LKreutzer — APPROVED** [review]

**Review by voisey — APPROVED** [review]

File: docs/docusaurus/versioned_docs/version-1.8.0/lte/extended_5g_sa_features.md [hunk 7]
```diff
@@ -7,7 +7,7 @@ original_id: extended_5g_sa_features
 
 # Extended Features of 5G SA FWA
 
-This document gives an overview of the Extended features added on top of the [integrated 5G SA FWA](https://docs.magmacore.org/docs/lte/integrated_5g_sa) (5G Standalone Architecture Fixed Wireless Access) such as
+This document gives an overview of the Extended features added on top of the [integrated 5G SA FWA](https://magma.github.io/magma/docs/next/lte/integrated_5G_sa) (5G Standalone Architecture Fixed Wireless Access) such as
```
Comment (Neudrino) [inline docs/docusaurus/versioned_docs/version-1.8.0/lte/extended_5g_sa_features.md]: Link gives 404 for me. Should be `5g_sa` with small `g` for me.

Comment (mpfirrmann) [reply docs/docusaurus/versioned_docs/version-1.8.0/lte/extended_5g_sa_features.md]: Yes, I fixed this today on master. Probably missed it during rebase.

**Review by Neudrino — APPROVED** [review]

**Pull request merged**